scss/bitstyles/utilities/margin/_.scss

```diff
@@ -18,6 +18,10 @@
     .#{$bitstyles-namespace}u-margin-#{$size}--#{$position} {
       margin-#{$position}: spacing($size);
     }
+
+    .u-margin-neg-#{$size}--#{$position} {
+      margin-#{$position}: -#{spacing($size)};
+    }
```

Comment:
I wonder why some classes here are namespaced, but others are not `#{$bitstyles-namespace}`...

Reply:
Yes! I added a ticket to go through and check all of them, instead of doing it here: https://github.com/bitcrowd/bitstyles/issues/469